greenboot-rs.spec

```diff
@@ -132,6 +146,9 @@ install -DpZm 0644 usr/lib/systemd/system/greenboot-healthcheck.service.d/10-net
 %{_unitdir}/greenboot-healthcheck.service.d/10-network-online.conf
 
 %changelog
+* Mon Sep 01 2025 Mario Cattamo <mcattamo@redhat.com> - 0.16.0-5
```

Comment:
You also need to increase the `Release: ` tag

Reply:
Done.